libsolidity/formal/SymbolicVariables.cpp

```diff
@@ -26,14 +26,30 @@ using namespace dev::solidity;
 
 SymbolicVariable::SymbolicVariable(
```

Comment:
It would be nice to call the other constructor, but then you cannot `solAssert(m_type`, can you?